test/replication/gh-5430-cluster-mvcc.result

```diff
@@ -136,7 +136,7 @@ test_run:cmd('delete server replica1')
  | ---
  | - true
  | ...
-test_run:cmd('stop server master')
+test_run:cmd('stop server master with signal=KILL')
```

Comment:
Why?

Reply:
Hmm, this was added by mistake. In other places of this patch I used `KILL` because of injection blocking graceful shutdown. Here is different, injection is reset before stopping the master.

Yet this test prints the below message from time to time.
```
[001] * [QA Notice]
[001] *
[001] * The server 'gh-5430-mvcc-master' does not stop during 5 seconds after the 15 (SIGTERM) signal.
[001] * Info: process 488354 [R (running); tarantool gh-5430-mvcc-master.lua <running>]
[001] * Sending SIGKILL...
```

As test is finished successfully even in the presence of the above message let's remove this change and investigate the issue in the scope of ticket https://github.com/tarantool/tarantool/issues/9617

Reply:
Turned out the issue with stopping master server was due to 2 factors. First is strange #9617 execution itself (so the ticket stays) which causes starting a lot of consoles on master during test. Second I forgot to cleanup `on_shutdown` trigger on console exit.